README.md

```diff
@@ -1,3 +1,37 @@
 # vmbuild
 
 Utilities for building IncludeOS VMs
+
+### Configure Settings
+
+```
+  conan config install https://github.com/includeos/conan_config.git
+```
+
+##### View remote and profiles installed
+
+* Remotes:
+```
+  conan remote list
+```
+You should have these:
+
+```
+  includeos: https://api.bintray.com/conan/includeos/includeos [Verify SSL: True]
+  includeos-test: https://api.bintray.com/conan/includeos/test-packages [Verify SSL: True]
+```
+
+* Profiles:
+
+```
+  conan profile list
+```
+
+
+### Build and run vmbuild
```

Comment:
This should probably point out to use the toolchain profiles ie apple-clang for mac and clang or gcc for linux